product_docs/docs/pgd/4/rel_notes/pgd_4.3.1-2_rel_notes.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: "Release notes for EDB Postgres Distributed version 4.3.1-2"
+navTitle: "Version 4.3.1-2"
+---
+
+EDB Postgres Distributed version 4.3.1-2 is a patch release of EDB Postgres Distributed 4, which includes bug fixes for issues identified in previous versions.
+
+It includes HARP 2.3.1 with bug fixes. If you are using HARP 2.3.0 or earlier, we recommend that you upgrade to HARP 2.3.1.
+
+!!! Note
```

Comment:
I am not sure whether this Note should exist here or not, kept it as it has been carried over since 4.3.0